views/partials/workshops/seminars.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="card">
+  <a href="{{mainLink}}" target="_blank">
+    <h5 class="card-title">{{name}}</h5>
+  </a>
+  <p class="card-text">{{description}}</p>
+  <a class="workshops-links" href={{link1}} target="_blank">{{link1}}</a>
+  <a class="workshops-links" href={{link2}} target="_blank">{{link2}}</a>
+  <a class="workshops-links" href={{link3}} target="_blank">{{link3}}</a>
+  <p class="card-text">{{additional}}</p>
+</div>
```

Comment:
Please, name this file in singular, without **s** - to match the other partials :)